package.json

```diff
@@ -19,58 +19,54 @@
     "@sentry/integrations": "^5.29.1",
     "axios": "^0.21.1",
     "babel-plugin-transform-decorators": "^6.24.1",
-    "core-js": "^3.7.0",
+    "core-js": "^3.8.0",
```

Comment:
For all of these, the package-lock file indicated we were actually using the newer version, so I just updated this to match.
That's why the package-lock has almost no changes in it.